### PR TITLE
add pull request template markdown file

### PR DESCRIPTION
**Pull request by lanebret85**

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+- [ ] docs
+
+## Checklist
+
+- [ ] code has been self reviewed
+- [ ] code runs without any errors
+- [ ] thorough testing has been implemented if adding feature
+- [ ] all tests pass
+
+### Thanks!
```
Comment (ttakahashi1591) [inline pull_request_template.md:20]: This looks great! Thank you for setting this up, it will really support with us having consistent PR's.

**Review by ttakahashi1591 — COMMENTED** [review]
This looks great! Thank you for setting this up, it will really support with us having consistent PR's.

File: pull_request_template.md (hunk 1 above)
Comment (aevans27) [reply pull_request_template.md:20]: Looks cool, we'll put it to good use.

**Review by aevans27 — COMMENTED** [review]
Looks cool, going to try it out on next pull

File: pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+## Description
+
+Please include a description of what was changed
+
+## Type of change
```
Comment (TLamb32) [inline pull_request_template.md:5]: I like this. Great idea. 👍

**Comment by TLamb32** [comment]
Looks great.. I'll go ahead and push through.

**Pull request merged**